pkg/metricscollector/prommetrics.go

```diff
@@ -177,6 +187,18 @@ func (p *PromMetrics) RecordScalerActive(namespace string, scaledObject string,
 	scalerActive.With(getLabels(namespace, scaledObject, scaler, scalerIndex, metric)).Set(float64(activeVal))
 }
 
+// RecordScalerPaused marks whether the current ScaledObject is paused.
+func RecordScalerPaused(namespace string, scaledObject string, active bool) {
```

Comment:
Is this function named RecordScaledObjectPaused better as there will be scaledjob paused metric? WDYT @zroubalik

Reply:
Good catch, you are right!

Reply:
@zroubalik also waiting for your reply on the above question.